SECURITY.md

```diff
@@ -0,0 +1,32 @@
+# Juno bug reporting and feature requests
+
+The Juno core development team uses GitHub to manage feature requests and bugs. This is done via GitHub Issues.
+
+## Triage and progress 🔜
+
+Issues added to GitHub will be triaged as they come in.
+
+Tracking of in-flight issues will be done through the Juno Core project board, but of course we reserve the right to not make a public issue if there is a security implication in doing so.
+
+## Feature request 🚀
+
+For a feature request, e.g. module inclusion, please make a GitHub issue. Clearly state your use case and what value it will bring to other users or developers on Juno.
+
+If it is something that can be handled by a param change, discuss it on Discord in the `#governance` channel, and consider a governance proposal.
+
+## Standard priority bug 🐛
+
+For a bug that is non-sensitive and/or operational in nature rather than a critical vulnerability, please add it as a GitHub issue.
+
+If it is not triaged in a couple of days, feel free to tag `@the-frey` or `@jakehartnell`.
+
+## Critical bug or security issue 💥
```

Comment:
I think we could do with a more professional disclosure policy. Here's a nice example from Tendermint: https://tendermint.com/security/

It's *really* important to set specific expectations around disclosure timeline. Otherwise feelings get hurt and bugs get disclosed in an uncoordinated way.

Reply:
Feel free to make a PR.

Reply:
Done - #124